serum-390/soen390-team16-godzilla
# Shipping frontend fix

# Description
This PR fixes the blocking call bug for the shipping  frontend
TODO: Describe your changes...

## Link to work item

TODO: Include a link to the work item E.g.: #1

## How to Test

TODO: Include steps to test your changes...
test by creating a docker image. navigate to the shipping page and try to schedule the shipping for a packaged order , then refresh the page. note that the order status must be packaged and the shipping date must be after the current date and before the order due date
